### DISCUSSION TITLE
Demo ROS script that implements the suturing task from 2021-2022 challenge?

### DISCUSSION THREAD

**Discussion by sunshineinsandiego** [surgical-robotics-ai/surgical_robotics_challenge#81] [Q&A]
Hi - Is there is a publicly available ROS demo script that implements the suturing task (similar to the gif [here](https://github.com/collaborative-robotics/surgical_robotics_challenge/blob/master/Media/figure_eight.gif))? Would like to see how this works. 

Perhaps the winning teams (CUHK, WPI, SDU) published their winning code somewhere?  Thanks!

**Comment by adnanmunawar** [comment]
Similar question answered here (https://github.com/collaborative-robotics/surgical_robotics_challenge/discussions/69#discussioncomment-5645954).

Only one of the winning teams was successful (partially) in Task 2 and they executed a fixed trajectory by using the error-free location of the PSMs and needle. That may not be very useful. As mentioned in the answer above.

> This video was created by tele-operating the PSMs via MTMs and not a script. If you have access to the dVRK MTMs you can do the same. If not, we can probably re-record this and share a ROS Bag with you with the trajectories for the PSMs.

**Reply by sunshineinsandiego** [reply]
Got it, thanks Adnan. Wrote a reply in that thread. Would love to take a look at that ROS bag.
